### DISCUSSION TITLE
Welcome to emby_pinyin Discussions!

### DISCUSSION THREAD

**Discussion by hisune** [hisune/emby_pinyin#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robinletian** [comment]
坐等docker 自动版。。。

**Reply by hisune** [reply]
有一种更好的方法，需要等emby支持新增内容的webhook通知后，可以做到新增了内容，自动通知emby_pinyin进行处理，但貌似官方对这个功能的开发不太上心
https://emby.media/community/index.php?/topic/97889-new-content-notification-webhook/
